tests/commands/autopart.py

```diff
@@ -360,5 +360,13 @@ def runTest(self):
         F38_TestCase.runTest(self)
         self.assert_parse_error("autopart --type=btrfs")
 
+class F41_TestCase(F38_TestCase):
+    def runTest(self):
+        F38_TestCase.runTest(self)
+        self.assert_parse("autopart --encrypted --luks-version=luks2-hw-opal --hw-passphrase=\"test\"",
```

Comment:
Also test with `--hw-passphrase` and `--passphrase` as different strings to ensure they aren't cross-populated.